evm/gateway.js

```diff
@@ -81,7 +81,7 @@ async function processCommand(config, chain, options) {
     printInfo('Contract name', contractName);
     printInfo('Contract address', gatewayAddress);
 
-    const gateway = new Contract(gatewayAddress, IGateway.abi, wallet);
+    const gateway = new Contract(gatewayAddress, AxelarGateway.abi, wallet);
```

Comment:
depending on gateway type (there's a connectionType field in config), can you initialize it to amplifier vs old gateway?

Reply:
done